pymc3/distributions/transforms.py

```diff
@@ -150,32 +150,32 @@ def __init__(self):
     def forward(self, x):
         #reverse cumsum
         x0 = x[:-1]
-        s = t.extra_ops.cumsum(x0[::-1])[::-1] + x[-1]
+        s = t.extra_ops.cumsum(x0[::-1], 0)[::-1] + x[-1]
         z = x0/s
         Km1 = x.shape[0] - 1
-        k = arange(Km1)
+        k = arange(Km1)[(slice(None), ) + (None, ) * (x.ndim - 1)]
```

Comment:
This ensures that k (and hence eq_share) has the same number of dimensions as x, which is necessary for it to be subtracted from logit(z).